doc/introduction/templates.rst

```diff
@@ -121,8 +118,84 @@ The following subroutines are available:
 Parameter initializations
 -------------------------
 
-Each trainable template has a dedicated function in the :mod:`pennylane.init` module, which generates a list of
-randomly initialized arrays for the trainable parameters.
+Each trainable template has dedicated functions in the :mod:`pennylane.init` module, which generate
+randomly initialized arrays for the trainable parameters. For example, :func:`random_layers_uniform` can
+be used together with the template :func:`RandomLayers`:
+
+.. code-block:: python
+
+    import pennylane as qml
+    from pennylane.templates import RandomLayers
+    from pennylane.init import random_layers_uniform
+
+    dev = qml.device('default.qubit', wires=3)
+
+    @qml.qnode(dev)
+    def circuit(weights):
+        RandomLayers(weights=weights, wires=[0, 2])
+        return qml.expval(qml.PauliZ(0))
+
+    init_pars = random_layers_uniform(n_layers=3, n_wires=2)
+    circuit(init_pars)
+
+Templates that take more than one parameter
+array require several initialization functions:
+
+.. code-block:: python
+
+    from pennylane.templates import Interferometer
+    from pennylane.init import (interferometer_theta_uniform,
+                                interferometer_phi_uniform,
+                                interferometer_varphi_normal)
+
+    dev = qml.device('default.gaussian', wires=3)
+
+    @qml.qnode(dev)
+    def circuit(theta, phi, varphi):
+        Interferometer(theta=theta, phi=phi, varphi=varphi, wires=[0, 2])
+        return qml.expval(qml.X(0))
+
+    init_theta = interferometer_theta_uniform(n_wires=2)
+    init_phi = interferometer_phi_uniform(n_wires=2)
+    init_varphi = interferometer_varphi_normal(n_wires=2)
+
+    circuit(init_theta, init_phi, init_varphi)
+
+
+For templates with multiple parameters, initializations that
+return a list of all parameter arrays at once are provided, and can
+be conveniently used in conjunction with the unpacking operator ``*``:
+
+.. code-block:: python
+
+    from pennylane.templates import Interferometer
+    from pennylane.init import interferometer_all
+
+    dev = qml.device('default.gaussian', wires=3)
+
+    @qml.qnode(dev)
+    def circuit(*pars):
+        Interferometer(*pars, wires=[0, 2])
+        return qml.expval(qml.X(0))
+
+    init_pars = interferometer_all(n_wires=2)
```

Comment:
No, this function chooses a strategy that is reasonable, which in this case is only uniform (since all are angles). If the user wants something special she has to code by hand.

Reply:
It's in the docstring actually